lib/src/gantt_default_week_header.dart

```diff
@@ -7,6 +7,17 @@ class GanttChartDefaultWeekHeader extends StatelessWidget {
   }) : super(key: key);
   final DateTime weekDate;
 
+  int get weekNumberWithinThisYear {
+    final numberWeek = (DateTime.utc(
+                    weekDate.year, weekDate.month, weekDate.day)
+                .difference(DateTime.utc(weekDate.year, 1, 1, 0, 0, 0, 0, 0))
+                .inDays /
+            7)
+        .ceil();
+
+    return numberWeek == 0 ? 1 : numberWeek;
```

Comment:
should be `return numberWeek + 1;`

Reply:
https://www.epochconverter.com/weeks/2023

Very Strange but based on link above it should not actually be incremented at all :)